CHANGELOG.md

```diff
@@ -0,0 +1,185 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
+
+## [Unreleased](https://github.com/voxpupuli/puppet-firewalld/tree/HEAD)
```

Comment:
You need to generate the changelog as part of the actual release PR and *after* you adjust the version in metadata.json

Reply:
even if it's the first time a Changelog is generated?